server/apps/archive/archive_composite.py

```diff
@@ -134,8 +134,11 @@ def update_link(self, package_id, assoc, delete=False):
         if not delete:
             two_way_links.append({'package': package_id})
 
+        # do update without changing etag/last_modified
         updates = self.get_item_update_data(item, two_way_links, delete)
-        get_resource_service(endpoint).patch(item_id, updates)
+        updates['_etag'] = item['_etag']
```

Comment:
it's single item, you update that same item, change there linked_in_packages, but keep the etag.
so if user (it might be other user, item can be locked etc) will save item it will still work. we can't reload that item for user who is editing content because someone else might pick it into a package imo